### PR TITLE
HLT Module for Saving DIGIs of Rechits for Pi0/Eta Calibration Stream [75X]

### PR DESCRIPTION
**Pull request by hardenbr**
74X request: https://github.com/cms-sw/cmssw/pull/8311

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

File: HLTrigger/special/src/HLTRechitsToDigis.cc [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+{ 
+   // do anything here that needs to be done at desctruction time
+   // (e.g. close files, deallocate resources etc.)
+}
+
+//
+// member functions
+//
+HLTRechitsToDigis::ecalRegion
+HLTRechitsToDigis::stringToRegion(const std::string & region) {
```
Comment (Martin-Grunewald) [inline HLTrigger/special/src/HLTRechitsToDigis.cc]: Should be static and should be const.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**